core/lib/generators/spree/install/templates/config/initializers/spree.rb.tt

```diff
@@ -93,3 +93,11 @@ Spree.user_class = <%= (options[:user_class].blank? ? "Spree::LegacyUser" : opti
 # just uncomment the following code and change it as you need.
 #
 # Spree::Model.whitelisted_ransackable_attributes << 'field'
+
+# Rules for avoiding to store the current path into session for redirects
+# When at least one rule is matched, the request path will not be stored
+# in session.
+# You can add your custom rules by uncommenting this line and changing
+# the class name:
+#
+# Spree::UserReturnToStorer.rules << 'Spree::UrlReturnToStorer::Rules::AuthenticationRule'
```

Comment:
I'd use an explicit example class here, like `Spree::UrlReturnToStorer::Rules::YourCustomRule`

Reply:
I used a real one so a developer interested in knowing how these rules work can quickly find that real existing module in the codebase and use it as a blueprint for their custom rules... or at least that's what I would do. 

Differently, one should look into the documentation or use their own means to find a real example, ending up on that very same file (it's the only rule we have so far) but probably spending more time and brain resources in the process.

Reply:
That makes sense!